test/inspect-provenance.test.ts

```diff
@@ -159,3 +159,88 @@ test('inspect --only-provenance for Pipfile', async (t) => {
     );
   });
 });
+
+
+const setupPyAppExpectedDependenciesProvenance = {
+  django: {
+    data: {
+      name: 'django',
+      version: '1.6.1',
+      labels: { provenance: 'setup.py:8' },
+    },
+    msg: 'django looks ok',
+  },
+  jinja2: {
+    data: {
+      name: 'jinja2',
+      version: '2.7.2',
+      labels: { provenance: 'setup.py:8' },
+    },
+    msg: 'jinja2 looks ok',
+  },
+  'python-etcd': {
+    data: {
+      name: 'python-etcd',
+      version: '0.4.5',
+      labels: { provenance: 'setup.py:8' },
+    },
+    msg: 'python-etcd is ok',
+  },
+  'django-select2': {
+    data: {
+      name: 'django-select2',
+      version: '6.0.1',
+      labels: { provenance: 'setup.py:8' },
```

Comment:
does the graphLib understand this already? Did not know that!

Reply:
It preserves labels. That's why I put the provenance info there.